gradle.properties

```diff
@@ -14,3 +14,27 @@ POM_LICENCE_DIST=repo
 
 POM_DEVELOPER_ID=square
 POM_DEVELOPER_NAME=Square, Inc.
+
+## Project-wide Gradle settings.
+#
+# For more details on how to configure your build environment visit
+# http://www.gradle.org/docs/current/userguide/build_environment.html
+#
+# The Gradle daemon aims to improve the startup and execution time of Gradle.
+# When set to true the Gradle daemon is to run the build.
+org.gradle.daemon=true
```

Comment:
The daemon is on by default in 3.0+ so this is redundant to specify. Plus i'm not sure how this affects the `--no-daemon` flag which we want to use on CI